### PR TITLE
Add Rspec generator

### PR DESCRIPTION
**Pull request by luathn**
I just added generator to install rspec: https://github.com/rspec/rspec-rails
Please check it out!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * Updates default ruby version of GitHub Actions install generator. ([@jamesglover][])
 * Simplify generated GitHub Actions install ([@jamesglover][])
 * Adds boring generator CLI ([@luathn][])
+* Adds Rspec generator. ([@luathn][])
```
Comment (abhaynikam) [inline CHANGELOG.md:7]:
```suggestion
* Adds RSpec generator. ([@luathn][])
```
As per official guide. Changed the case here to match it. Could you update at other places as well?

File: lib/generators/boring/rspec/install/install_generator.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds rspec to the application"
+
+      class_option :version, type: :string, aliases: "-v",
+                             desc: "Tell us the gem version which you want to use for the application"
+
+      def add_rspec_gem
+        say "Adding rspec gem", :green
+        rspec_gem_content = generate_gem_content
+        append_to_file "Gemfile", rspec_gem_content
```
Comment (abhaynikam) [inline lib/generators/boring/rspec/install/install_generator.rb:14]: The gem content would add another `group :development, :test do` to gem file. Instead can you try to use `insert_into_file` with regex to match `group :development, :test do` and add it gem within existing group?

**Review by abhaynikam — COMMENTED** [review]
@luathn Thanks ❤️ . PR looks awesome to me. Some minor nitpicks and suggestion to improve the generator overall.

Whenever I think of using rspec in the project I would also use 
- `factory-bot`. We can add this as separate generate but add an option to invoke the generator from here if option is given
- Configure the scaffold generators to generate the test cases for rspec instead of minitest.
- Configuration in the test helper to run rspec specs.
- Some configuration to print minimal description with proper color config. I don't remember it but I'll make I can point you in that direction.
- Example test cases? Maybe. 

@luathn Not necessary we handle everything within this PR. You can raise multiple PRs as well. 😊

File: lib/generators/boring/rspec/install/install_generator.rb (hunk 2 above)
Comment (luathn) [reply lib/generators/boring/rspec/install/install_generator.rb:14]: It's great idea. I'll do it when I have time :100:

**Comment by abhaynikam** [comment]
@luathn Soft ping. Are you still interested in working on this generator? Let me know if you have bandwidth to complete it. RSpec generator surely adds value to the gem and most people(include me) would use it in the projects 😊

**Comment by luathn** [comment]
@abhaynikam
Sorry I'm a bit slow. Lately I've been quite busy. I will come back to this Pr next week

**Comment by abhaynikam** [comment]
Added rspec generator in #63

**Pull request closed without merging**